onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -151,7 +156,6 @@ private long getReminderMin(String tripId, String stopId) {
      * Checks arrivals from given ObaArrivalInfoResponse
      *
      * @param response arrival information
-     * @param tripId
```

Comment:
Looks like the Javadoc for `tripId` was removed - could you please re-add with short description?